plugins/onex/skills/epic-team/SKILL.md

```
@@ -117,29 +124,40 @@ Task(
 )
 ```
 
-## Dispatch: Worker per Repo
+## Dispatch: Ticket-Pipeline per Ticket (Direct Dispatch Pattern)
+
+For each ticket in a wave, dispatch ticket-pipeline as a Task() from the team-lead session:
 
 ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)